prow/tide/tide.go

```diff
@@ -739,10 +740,20 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 	for i, pr := range prs {
 		backoff := time.Second * 4
 		log := sp.log.WithFields(pr.logFields())
+		mergeMethod := c.ca.Config().Tide.MergeMethod(sp.org, sp.repo)
+		if squashLabel := c.ca.Config().Tide.SquashLabel; squashLabel != "" {
+			il, err := c.ghc.GetIssueLabels(sp.org, sp.repo, int(pr.Number))
```

Comment:
We don't need to query the issue labels. These are already provided by the `pr.Labels` field.